process-automation/latest/model/connectors/content.md

```diff
@@ -597,6 +597,48 @@ The output parameters from declaring a record are:
 | --------- | ---- | ----------- |
 | response | JSON | *Optional.* The response from the call declaring the record. |
 
+## Security Marks
+
+The **ADD_SECURITY_MARKS** action is used to add security marks to specific content.  
+To add security marks in Process Automation you must have configured security marks in Governance Services. For information on security marks in Governance Services, see [Security Marks and Classification
+]({% link governance-services/latest/using/smc.md %})
+
+The input parameters from add security marks are:
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| file | file | *Required.* ACS file to be updated. File variables are initialised by content actions or content events in triggers. E.g: 'Initialise a file variable by mapping it to the output of the generate document task. |
+| folder | folder | *Required.* ACS folder to be updated. Folder variables are initialised by content actions or content events in triggers. |
+| path | String | *Required.* Location or relative path of the node to be updated. For example a location path could be `/app:company_home/app:user_homes/cm:hruser` and a relative path could be `/User Homes/hruser` (by default the relative path is from Company Home). |
+| nodeId | String | *Required.* Node ID of the file or folder in ACS. Example:\n'a6a977a6-c728-4038-8dbc-d914c4d8cfb3. |
+| securityGroupName | String | *Required.* Security group that contains the security marks to be assigned. E.g: 'PII'. |
+| securityMarks | array | *Required.* Array including the name of the security marks to be added. |
+
+The output parameters from add security marks are:
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| response | json | *Optional.* Response for the calls. |
```

Comment:
No need to use *Optional* here as the developer cannot control output parameters.